sdk/core/Azure.Core.Extensions/samples/Startup.cs

```diff
@@ -48,11 +50,14 @@ public void ConfigureServices(IServiceCollection services)
 
                 // Advanced configure global defaults
                 builder.ConfigureDefaults((options, provider) =>  options.AddPolicy(HttpPipelinePosition.PerCall, provider.GetService<DependencyInjectionEnabledPolicy>()));
+
+                builder.AddBlobServiceClient(Configuration.GetSection("Storage"))
```

Comment:
Need to add the updated config file?

Reply:
Will we need to pass the connection string to the `AddBlobServiceClient` call? Or can that be part of the `Storage` configuration section?

Reply:
Will we need to pass the connection string to the `AddBlobServiceClient` call? Or can that be part of the `Storage` configuration section?

Reply:
The connection string can be a part of the configuration.